### ISSUE TITLE
bil-CV: Options are defined after class is loaded

### ISSUE THREAD

**Issue by forquare** [forquare/LaTeX-Document-Classes#1]
All font size options are processed, but no option is formally declared. 
Ok, no problem, let's just swap that and process the options after declaring. 
What does option 12pt do? It passes the option 12pt to the base class, which is article here. But, and here comes the catch, the class is already loaded, with an empty option set. So, once again, useful information is lost, no matter which option you give, the document will be typeset with a 10 pt font.

SO, to sum it up, we need to declare valid options, take the user specified options and process them. Load the class, if needed, we can pass options to the class, but we have to do it before loading. 

From [LaTeX Community](http://latex-community.org/forum/viewtopic.php?f=4&t=26245#p89615)

**Comment by forquare** [comment]
Fixed in [7352d27](https://github.com/forquare/LaTeX-Document-Classes/commit/7352d27e0954d2eb258014ba6f48c62068579f17)
